// containers/catalog/index.js
import React from 'react';
import { connect } from 'react-redux';
import { Link } from 'react-router';
import R from 'ramda';

import {
    fetchGoods,
    fetchCategories,
    addGoodToBasket
} from 'actions';

import {
    getGoods,
    goodsUniqSetId,
    getArrArrsGoodsByCategory
} from 'selectors';

import Sidebar from 'components/sidebar';

class Catalog extends React.Component {
    constructor(props) {
        super(props);
        this.state = {};
        this.onChangeInput = this.onChangeInput.bind(this);
        this.addGoodToBasketAndCleanState = this.addGoodToBasketAndCleanState.bind(this);
    }
    componentDidMount()
    {
        this.props.fetchGoods();
    }
    addGoodToBasketAndCleanState(goods) {
        this.props.addGoodToBasket(goods);
        this.state = {};
    }
    onChangeInput(e) {
        this.setState({
            [e.target.name] : e.target.value
            });
        e.preventDefault();
    }
    renderColorBlock(good, ind) {
        const classColor = (colorId) => {
            switch(colorId) {
                case '1' :
                    return 'red';
                case '2' :
                    return 'orange';
                case '3' :
                    return 'yellow';
                case '4' :
                    return 'green';
                case '5' :
                    return 'cyan';
                case '6' :
                    return 'blue';
                case '7' :
                    return 'purple';
                default :
                    return '';
            }
        };
        return (
            <div
                className="form-group formGoodColor__group"
                key={ind}
            >
                <label
                    htmlFor={good.id}
                    className={`col-xs-6 control-label formGoodColor__label ${classColor(good.color.id)}`}
                />
                <input
                    type="text"
                    placeholder="0"
                    name={good.id}
                    className="form-control"
                    value={this.state[good.id] || ''}
                    onChange={this.onChangeInput}
                />
            </div>
        )
    }
    renderBallon(arrArrsGoods) {
        return arrArrsGoods.map( (arrGoods, ind) => {
            const arrColor = arrGoods.map( (good, ind) => this.renderColorBlock(good, ind));
            return (
                <div
                    className='col-sm-6'
                    key={ind}
                >
                    <div className="panel panel-default">
                        <div className="panel-heading panel-heading_small">
                            <div className="panel-title">
                                <Link to={`/${arrGoods[0].category}/${arrGoods[0].id}`}>
                                    {arrGoods[0].name}
                                </Link>
                            </div>
                        </div>
                        <div className="panel-body panel-body_small">
                            <ul className="list-group list-group_woMarginBottom">
                                <li className="list-group-item list-group-item_small">
                                    <img src={arrGoods[0].image} title={arrGoods[0].name} alt={arrGoods[0].name} className='list-group-item__image'/>
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Размер:</span> {arrGoods[0].shape.desc}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <div className="small">Цвет и количество:</div>
                                    <div
                                        className="list-group list-group_woMarginBottom"
                                    >
                                        <div className="list-group-item list-group-item_small">
                                            <form className="form-horizontal formGoodColor">
                                                { arrColor }
                                            </form>
                                        </div>
                                        <div className="list-group-item list-group-item_small">
                                            <div
                                                className="btn btn-primary btn-block"
                                                onClick={() => this.addGoodToBasketAndCleanState(this.state)}
                                            >
                                                В корзину
                                            </div>
                                        </div>
                                    </div>
                                </li>
                                <li className="list-group-item  list-group-item_small">
                                    <span className="small">Рисунок:</span> {arrGoods[0].print.desc}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Поверхность:</span> {arrGoods[0].cover.desc}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Цена:</span> {arrGoods[0].price}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Описание:</span> { R.take(40, arrGoods[0].description) }...
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
            )
        } );
    }
    renderGoods() {
        const arrArrsSortGoodsBySetId = goodsUniqSetId(this.props.goods);
        const arrArrsHeLatexByCategory = getArrArrsGoodsByCategory(arrArrsSortGoodsBySetId, '1');
        const arrArrsAirLatexByCategory = getArrArrsGoodsByCategory(arrArrsSortGoodsBySetId, '2');
        return R.append( this.renderBallon(arrArrsHeLatexByCategory), this.renderBallon(arrArrsAirLatexByCategory) );
    }

    render() {
        return (
            <div className='col-xs-12'>
                <div className="row">
                    <div className="col-sm-4">
                        <Sidebar />
                    </div>
                    <div className="col-sm-8">
                        <div className="row">
                            { this.renderGoods() }
                        </div>
                    </div>
                </div>
            </div>
        )
    }
}
const mapStateToProps = state => ({
    goods : getGoods(state),
    goodsPage : state.goodsPage.ids
});
const mapDispatchToProps =({
    fetchGoods,
    fetchCategories,
    addGoodToBasket
});

export default connect(mapStateToProps, mapDispatchToProps)(Catalog)
