// containers/basket/index.js
import React from 'react';
import { connect } from 'react-redux';
import R from 'ramda';

import {
    getGoodsInBasket,
    amount,
    coast
} from 'selectors';

import {
    deleteFromBasket
} from 'actions';

class Basket extends React.Component {
    renderTable({name, shape, color, cover, print, price, amount, image, id, category}, ind) {
        return (
            <tr
                key={ind}
            >
                <td>{name}, {shape.desc}, {color.desc}, {cover.desc}, {print.desc}</td>
                <td>{amount}</td>
                <td>{price}</td>
                <td>
                    <div
                        className="btn btn-default"
                        onClick={ () => { this.props.deleteGood(id) } }
                    >
                        <span className="glyphicon glyphicon-trash" />
                    </div>
                </td>
            </tr>
        )
    }
    render() {
        const BasketEmpty = R.isEmpty(this.props.goodsInBasket);
        return (
            <div className="col-md-8 col-md-offset-2">
                <div className='panel panel-default'>
                    <div className="panel-heading">
                        <div className="panel-title">
                            Корзина
                        </div>
                    </div>
                    <div className="panel-body">
                        { BasketEmpty && <div>Ваша корзина пустая</div> }
                        { R.not(BasketEmpty) &&
                            <table className="table table-hover">
                                <thead>
                                <tr>
                                    <th>Товар</th>
                                    <th>шт.</th>
                                    <th>руб./шт.</th>
                                    <th/>
                                </tr>
                                </thead>
                                <tbody>
                                { this.props.goodsInBasket.map( (good, ind) => this.renderTable(good, ind))}
                                </tbody>
                                <tfoot>
                                <tr>
                                    <td/>
                                    <td>
                                        <div>
                                            Общее количесто:
                                        </div>
                                        { this.props.amountAllGoods } шт.
                                    </td>
                                    <td>
                                        <div>
                                            Общая стоимость:
                                        </div>
                                        { this.props.coastAllGoods } руб.
                                    </td>
                                    <td/>
                                </tr>
                                </tfoot>
                            </table>
                        }
                    </div>
                </div>
                <button
                    className="btn btn-primary btn-block"
                >
                    Заказать
                </button>
            </div>
        )
    }
}
const mapStateToProps = state => ({
    goodsInBasket : getGoodsInBasket(state),
    amountAllGoods : amount(state),
    coastAllGoods : coast(state)
});
const mapDispatchToProps = {
    deleteGood : deleteFromBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(Basket)
