// containers/airBallon/index.js
import React from 'react';
import { connect } from 'react-redux';

import {
    getGoods,
    goodsUniqSetId
} from 'selectors';

class AirBallon extends React.Component {
    constructor(props) {
        super(props);
        this.state = {};
        this.onChangeInput = this.onChangeInput.bind(this);
        this.addGoodToBasketAndCleanState = this.addGoodToBasketAndCleanState.bind(this);
    }
    addGoodToBasketAndCleanState(goods) {
        this.props.addGoodToBasket(goods);
        this.state = {};
    }
    onChangeInput(e) {
        this.setState({
            [e.target.name] : e.target.value
        });
        e.preventDefault();
    }

    renderColorBlock(good, ind) {
        const classColor = (colorId) => {
            switch(colorId) {
                case '1' :
                    return 'red';
                case '2' :
                    return 'orange';
                case '3' :
                    return 'yellow';
                case '4' :
                    return 'green';
                case '5' :
                    return 'cyan';
                case '6' :
                    return 'blue';
                case '7' :
                    return 'purple';
                default :
                    return '';
            }
        };
        return (
            <div
                className="form-group formGoodColor__group"
                key={ind}
            >
                <label
                    htmlFor={good.id}
                    className={`col-xs-6 control-label formGoodColor__label ${classColor(good.color.id)}`}
                />
                <input
                    type="text"
                    placeholder="0"
                    name={good.id}
                    className="form-control"
                    value={this.state[good.id] || ''}
                    onChange={this.onChangeInput}
                />
            </div>
        )
    }

    renderBallon(arrArrsGoods) {
        return arrArrsGoods.map( (arrGoods, ind) => {
            const arrColor = arrGoods.map( (good, ind) => this.renderColorBlock(good, ind));
            return (
                <div
                    className='col-sm-8 col-sm-offset-2'
                    key={ind}
                >
                    <div className="panel panel-default">
                        <div className="panel-heading panel-heading_small">
                            <div className="panel-title">
                                {arrGoods[0].name}
                            </div>
                        </div>
                        <div className="panel-body panel-body_small">
                            <ul className="list-group list-group_woMarginBottom">
                                <li className="list-group-item list-group-item_small">
                                    <img src={arrGoods[0].image} title={arrGoods[0].name} alt={arrGoods[0].name} className='list-group-item__image'/>
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Размер:</span> {arrGoods[0].shape.desc}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <div className="small">Цвет и количество:</div>
                                    <div
                                        className="list-group list-group_woMarginBottom"
                                    >
                                        <div className="list-group-item list-group-item_small">
                                            <form className="form-horizontal formGoodColor">
                                                { arrColor }
                                            </form>
                                        </div>
                                        <div className="list-group-item list-group-item_small">
                                            <div
                                                className="btn btn-primary btn-block"
                                                onClick={() => this.addGoodToBasketAndCleanState(this.state)}
                                            >
                                                В корзину
                                            </div>
                                        </div>
                                    </div>
                                </li>
                                <li className="list-group-item  list-group-item_small">
                                    <span className="small">Рисунок:</span> {arrGoods[0].print.desc}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Поверхность:</span> {arrGoods[0].cover.desc}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Цена:</span> {arrGoods[0].price}
                                </li>
                                <li className="list-group-item list-group-item_small">
                                    <span className="small">Описание:</span> {arrGoods[0].description}
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
            )
        } );
    }

    render() {
        const arrGoodsOneSetId = goodsUniqSetId(this.props.goods).filter(arr => arr.find(item => item.id === this.props.params.id) );
        return (
            <div>
                { this.renderBallon(arrGoodsOneSetId) }
            </div>
        )
    }
};
const mapStateToProps = state => ({
    goods : getGoods(state)
});

export default connect(mapStateToProps, null)(AirBallon)
