// components/categories/index.js
import React from 'react';
import { connect } from 'react-redux';
import { Link, withRouter } from 'react-router';
import classNames from 'classnames';
import { compose } from 'redux';
import R from 'ramda';

import {
    getActiveCategoriesId,
    getCategories,
    checkAllCategories
} from 'selectors';

import {
    fetchCategories,
} from 'actions';

class Categories extends React.Component {
    componentDidMount() {
        this.props.fetchCategories();
    }
    renderMenuItem({ id, name }, ind) {

        const compareActiveCategoryId = () => {
            return R.equals(id, this.props.activeCategoryId)
        };
        const listClass = classNames({
            active : compareActiveCategoryId()
        });

        return (
            <li
                key={ind}
                className={listClass} >
                <Link
                    to={`/category/${id}`}>
                    { name }
                </Link>
            </li>
        )
    }
    renderMenu() {
        const compareActiveCategoryId = () => {
            return R.equals(this.props.allCategories, '/allCategories')
        };
        const listClass = classNames({
            active : compareActiveCategoryId()
        });
        return (
            <div className="collapse navbar-collapse" id="myMenu">
                <ul className="nav navbar-nav">
                    <li
                        className={listClass} >
                        <Link
                            to={'/allCategories'}>
                            Весь каталог
                        </Link>
                    </li>
                    { this.props.categories.map( (category, ind) => this.renderMenuItem(category, ind) ) }
                </ul>
            </div>
        )
    }
    render() {
        return (
            <nav className="navbar navbar-default menu" role="navigation">
                <div className="container-fluid">
                    <div className="navbar-header ">
                        <button type="button" className="navbar-toggle" data-toggle="collapse" data-target="#myMenu">
                            <span className="sr-only">Toggle navigation</span>
                            <span className="icon-bar" />
                            <span className="icon-bar" />
                            <span className="icon-bar" />
                        </button>
                        <Link
                            className="navbar-brand"
                            to={"/"}
                        >
                            <img className='navbar-brand__img' src="/images/logo_text.svg" alt="логотип" title='логотип'/>
                        </Link>
                    </div>
                    { this.renderMenu() }
                </div>
            </nav>
        )
    }
}
const mapStateToProps = (state, ownProps) => ({
    categories : getCategories(state),
    activeCategoryId : getActiveCategoriesId(ownProps),
    allCategories : checkAllCategories(ownProps)
});
const mapDispatchToProps = ({
    fetchCategories
});
export default compose(
    withRouter,
    connect(mapStateToProps,mapDispatchToProps)
)(Categories)
