// api/mockGoods.js
export default [
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "1",
        price : "40 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_09_1_0_1_0",
        image : "/uploads/1_09_1_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "1",
        price : "40 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_09_2_0_1_0",
        image : "/uploads/1_09_2_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "3",
            desc : "желтый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "1",
        price : "40 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_09_3_0_1_0",
        image : "/uploads/1_09_3_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "4",
            desc : "зеленый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "1",
        price : "40 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_09_4_0_1_0",
        image : "/uploads/1_09_4_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "5",
            desc : "голубой",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "1",
        price : "40 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_09_5_0_1_0",
        image : "/uploads/1_09_5_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "6",
            desc : "синий",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "1",
        price : "40 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_09_6_0_1_0",
        image : "/uploads/1_09_6_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "1",
        price : "40 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_09_7_0_1_0",
        image : "/uploads/1_09_7_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "2",
        price : "48 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_12_1_0_1_0",
        image : "/uploads/1_12_1_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "2",
        price : "48 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_12_2_0_1_0",
        image : "/uploads/1_12_2_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "3",
            desc : "желтый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "2",
        price : "48 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_12_3_0_1_0",
        image : "/uploads/1_12_3_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "4",
            desc : "зеленый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "2",
        price : "48 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_12_4_0_1_0",
        image : "/uploads/1_12_4_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "5",
            desc : "голубой",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "2",
        price : "48 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_12_5_0_1_0",
        image : "/uploads/1_12_5_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "2",
        price : "48 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_12_7_0_1_0",
        image : "/uploads/1_12_7_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "3",
        price : "55 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_14_1_0_1_0",
        image : "/uploads/1_14_1_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "3",
        price : "55 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_14_2_0_1_0",
        image : "/uploads/1_14_2_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "3",
            desc : "желтый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "3",
        price : "55 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_14_3_0_1_0",
        image : "/uploads/1_14_3_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "4",
            desc : "зеленый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "3",
        price : "55 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_14_4_0_1_0",
        image : "/uploads/1_14_4_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "5",
            desc : "голубой",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "3",
        price : "55 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_14_5_0_1_0",
        image : "/uploads/1_14_5_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "6",
            desc : "синий",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "3",
        price : "55 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_14_6_0_1_0",
        image : "/uploads/1_14_6_0_1_0.jpg",
    },
    {name : "Шарик латексный с гелием",
        category : "1",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "3",
        price : "55 руб./шт",
        description : "Одноцветный шар из натурального латекса. Предназначенный для использования в оформлении.",
        id : "1_14_7_0_1_0",
        image : "/uploads/1_14_7_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "4",
        price : "30 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_09_1_0_1_0",
        image : "/uploads/2_09_1_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "4",
        price : "30 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_09_2_0_1_0",
        image : "/uploads/2_09_2_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "3",
            desc : "желтый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "4",
        price : "30 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_09_3_0_1_0",
        image : "/uploads/2_09_3_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "4",
            desc : "зеленый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "4",
        price : "30 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_09_4_0_1_0",
        image : "/uploads/2_09_4_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "5",
            desc : "голубой",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "4",
        price : "30 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_09_5_0_1_0",
        image : "/uploads/2_09_5_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "6",
            desc : "синий",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "4",
        price : "30 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_09_6_0_1_0",
        image : "/uploads/2_09_6_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "09",
            desc : "9 дюймов (22 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "4",
        price : "30 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_09_7_0_1_0",
        image : "/uploads/2_09_7_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "5",
        price : "35 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_12_1_0_1_0",
        image : "/uploads/2_12_1_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "5",
        price : "35 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_12_2_0_1_0",
        image : "/uploads/2_12_2_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "3",
            desc : "желтый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "5",
        price : "35 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_12_3_0_1_0",
        image : "/uploads/2_12_3_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "6",
        price : "35 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_12_1_0_2_0",
        image : "/uploads/2_12_1_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "6",
        price : "35 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_12_2_0_2_0",
        image : "/uploads/2_12_2_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "4",
            desc : "зеленый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "6",
        price : "35 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_12_4_0_2_0",
        image : "/uploads/2_12_4_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "12",
            desc : "12 дюймов (30 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "6",
        price : "35 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_12_7_0_2_0",
        image : "/uploads/2_12_7_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_1_0_1_0",
        image : "/uploads/2_14_1_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_2_0_1_0",
        image : "/uploads/2_14_2_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "3",
            desc : "желтый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_3_0_1_0",
        image : "/uploads/2_14_3_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "5",
            desc : "голубой",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_5_0_1_0",
        image : "/uploads/2_14_5_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "1",
            desc : "матовая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_7_0_1_0",
        image : "/uploads/2_14_7_0_1_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_1_0_2_0",
        image : "/uploads/2_14_1_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_2_0_2_0",
        image : "/uploads/2_14_2_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "3",
            desc : "желтый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_3_0_2_0",
        image : "/uploads/2_14_3_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "4",
            desc : "зеленый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_4_0_2_0",
        image : "/uploads/2_14_4_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "5",
            desc : "голубой",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_5_0_2_0",
        image : "/uploads/2_14_5_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "6",
            desc : "синий",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_6_0_2_0",
        image : "/uploads/2_14_6_0_2_0.jpg",
    },
    {name : "Шарик латексный с воздухом",
        category : "2",
        shape : {id : "14",
            desc : "14 дюймов (32 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "2",
            desc : "глянцевая",
        },
        option : "false",
        setId : "7",
        price : "37 руб./шт.",
        description : "Одноцветный шар из натурального латекса. Тип - пастель - матовый оттенок цвета. Предназначенный для использования в оформлении.",
        id : "2_14_7_0_2_0",
        image : "/uploads/2_14_7_0_2_0.jpg",
    },
    {name : "Шарик металлизированный с гелием",
        category : "3",
        shape : {id : "18",
            desc : "18 дюймов (46 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "8",
        price : "80 руб./шт.",
        description : "Шарик металлизированный, небольшого размера.",
        id : "3_18_1_0_3_0",
        image : "/uploads/3_18_1_0_3_0.jpg",
    },
    {name : "Шарик металлизированный с гелием",
        category : "3",
        shape : {id : "18",
            desc : "18 дюймов (46 см)",
        },
        color : {id : "4",
            desc : "зеленый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "8",
        price : "80 руб./шт.",
        description : "Шарик металлизированный, небольшого размера.",
        id : "3_18_4_0_3_0",
        image : "/uploads/3_18_4_0_3_0.jpg",
    },
    {name : "Шарик металлизированный с гелием",
        category : "3",
        shape : {id : "36",
            desc : "36 дюймов (92 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "9",
        price : "120 руб./шт.",
        description : "Шарик металлизированный, небольшого размера.",
        id : "3_36_1_0_3_0",
        image : "/uploads/3_36_1_0_3_0.jpg",
    },
    {name : "Шарик металлизированный с гелием",
        category : "3",
        shape : {id : "36",
            desc : "36 дюймов (92 см)",
        },
        color : {id : "6",
            desc : "синий",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "9",
        price : "120 руб./шт.",
        description : "Шарик металлизированный, небольшого размера.",
        id : "3_36_6_0_3_0",
        image : "/uploads/3_36_6_0_3_0.jpg",
    },
    {name : "Шарик металлизированный с гелием",
        category : "3",
        shape : {id : "36",
            desc : "36 дюймов (92 см)",
        },
        color : {id : "7",
            desc : "фиолетовый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "9",
        price : "120 руб./шт.",
        description : "Шарик металлизированный, небольшого размера.",
        id : "3_36_7_0_3_0",
        image : "/uploads/3_36_7_0_3_0.jpg",
    },
    {name : "Шарик металлизированный с воздухом",
        category : "4",
        shape : {id : "36",
            desc : "36 дюймов (92 см)",
        },
        color : {id : "2",
            desc : "оранжевый",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "10",
        price : "100 руб./шт.",
        description : "Шарик металлизированный, небольшого размера.",
        id : "4_36_2_0_3_0",
        image : "/uploads/4_36_2_0_3_0.jpg",
    },
    {name : "Хлопушки",
        category : "5",
        shape : {id : "36",
            desc : "36 дюймов (92 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "11",
        price : "56 руб.",
        description : "Набор хлопушек.",
        id : "5_36_1_0_3_0",
        image : "/uploads/5_36_1_0_3_0.jpg",
    },
    {name : "Гирлянда",
        category : "5",
        shape : {id : "36",
            desc : "36 дюймов (92 см)",
        },
        color : {id : "1",
            desc : "красный",
        },
        print : {id : "0",
            desc : "без рисунка",
        },
        cover : {id : "3",
            desc : "металлизированная",
        },
        option : "false",
        setId : "12",
        price : "24 руб.",
        description : "Гирлянда длинная, разноцветная.",
        id : "5_36_1_0_3_0",
        image : "/uploads/5_36_1_0_3_0.jpg",
    }
]
